docs/src/test/scala/docs/http/scaladsl/HttpServerExampleSpec.scala

```diff
@@ -525,14 +524,18 @@ class HttpServerExampleSpec extends WordSpec with Matchers
 
     object WebServer {
 
-      case class Bid(userId: String, bid: Int)
+      case class Bid(userId: String, offer: Int)
       case object GetBids
       case class Bids(bids: List[Bid])
 
-      class Auction extends Actor {
+      class Auction extends Actor with ActorLogging {
```

Comment:
Same here, let's include this snippet in the example. Remove surrounding doc marker